src/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,103 @@
+
+Scaling must use the same instance type and size for each node:
+
+-  `C5 instances` for each WEB node
+-  `M5 or M5n instances` for each Core node
+
+### Core tier scaling
+
+There are 3 nodes (core nodes) for the database and included services, such as ElasticSearch, MariaDB, and Redis. When the core tier approaches capacity, the only way to scale is to increase the server size, such as boosting the CPU power and memory. Capacity is limited to the size of the node that is available. Because the database cluster is designed for high availability, you cannot scale horizontally in a reliable way with the technologies used.
+
+![Core tier scaling]
+
+Consider an example that the core node instance type is _m5.2xlarge_ with 32Gb RAM. A service, such as the database, uses a considerable amount of memory (30Gb). Scaling to the next available instance size _m5.4xlarge_ provides 64Gb RAM, double the memory, and accommodates the growing needs of the database.
+
+Further optimizing performance includes routing traffic. You can route traffic based on the node type. By default the db node is isolated from the web traffic, so, as an example, you can choose to serve web traffic on the db node.
+
+### Web tier scaling
+
+There are 3 nodes for php-fpm and manages web traffic. In addition to vertical scaling by increasing power and memory, the web tier can scale horizontally by adding extra web servers to an existing cluster when constricted at the PHP level.
+
+![Web tier scaling]
+
+This complements the vertical scaling provided by the database tier.
+
+Consider an example that the web node instance type is _C5.2xlarge with 8 CPU and 16Gb RAM_. The number of requests to the site increased greatly. You can add an additional C5.2xlarge node to handle the increase in php-fpm processes or you can change each instance type to _C5.4xlarge with 16 CPU and 32Gb RAM_. Adding an additional node reduces the risk of insufficient surge capacity.
+
+## Project structure
+
+Minimally, Pro projects with the Scaling architecture have 6 nodes available.
+
+-  3 web nodes c5.2xlarge (8 CPU, 16 Gb RAM)
+-  3 core nodes m5.2xlarge (8 CPU, 32 Gb RAM)
+
+Each project is unique, however, and requires performance monitoring to properly analyze resource management. Each account includes the New Relic Infrastructure service, which automatically connects with the application data and performance analytics to provide dynamic server monitoring. See [New Relic services][nri].
+
+### SSH access
+
+Certain files and logs, such as the `/app/<project-id>/var/log` directory, are not shared between nodes. Each node has a unique SSH access. You can not use the Magento Cloud CLI to log in to the core or web nodes, but you can find the [SSH Access list] in the Project Web UI. The SSH Access 1 to 3 are Core nodes and 4 to _n_ are web nodes.
```

Comment:
```suggestion
Certain files and logs, such as the `/app/<project-id>/var/log` directory, are not shared between nodes. Each node has a unique SSH access. You cannot use the Magento Cloud CLI to log in to the core or web nodes, but you can find the [SSH Access list] in the Project Web UI. In the list, SSH Access 1 to 3 are for accessing Core nodes, and 4 to _n_ are for accessing web nodes.
```

Reply:
I do not use `cannot`. Often times it is necessary to bold the NOT so that it draws attention.